src/Organization/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import QueryWrapper from "../QueryWrapper"
+import organizationQuery from "./OrganizationQuery"
+import Issue from "./issue"
+import { render } from 'react-dom';
+
+// adding a coment for the pull request
+
+type OrganizationProps = {
+  repositories: {nodes: Repository[]},
+  login: string,
+}
+
+type Repository = {
+  issues: {nodes: IssueProps[]},
+  name: string,
+}
+
+export type IssueProps = {
+  created_at: Date,
+  url: string,
+  state: string,
+  title: string,
+  author: {
+    login: string
+  }
+}
+
+class Organization extends Component<OrganizationProps> {
+  renderIssue = (issues: Array<IssueProps>) => {
+    return(
+      issues.map(issue =>{
+        return <Issue {...issue} />
+      })
+    )
+  }
+  
+  render() {
+    return (
+      <div className="issueList">
+        {this.renderIssue(this.props.repositories.nodes[0].issues.nodes)}
```

Comment:
This function appears to be rendering a list of issues. Consider renaming it to `renderIssues`.